src/string_utils.h

```diff
@@ -104,4 +104,18 @@ static bool try_stoull(unsigned long long& val, const std::string& str, std::siz
     return false;
 }
 
+static float parse_float(const std::string& s, std::size_t* float_len = nullptr){
+    std::stringstream ss(s);
+    ss.imbue(std::locale("C"));
```

Comment:
Actually, `std::locale::classic()` could be preferred maybe

Reply:
The two are equivalent, but you're right, this way we avoid a constructor call.